command/csv/field-object.csv

```diff
@@ -0,0 +1,27 @@
+FIELD NAME,ATTRIBUTES,DATA TYPE,DESCRIPTION

```

Comment:
Is this a reference of fields, @deviantintegral ?

Reply:
This is a CSV copied out from mpx's documentation. See `bin/console help mpx:create-data-service` for details - it's how I bootstraped the Field class in this PR.